### PR TITLE
Fixes issue where DISABLE_AUTHENTICATION was always set to True

### PR DESCRIPTION
**Pull request by pkajaba**

### CODE REVIEW

File: f8a_jobs/defaults.py [hunk 1]
```diff
@@ -9,8 +9,14 @@
...
-DISABLE_AUTHENTICATION = bool(os.environ.get('DISABLE_AUTHENTICATION', False))
 GITHUB_CONSUMER_KEY = os.environ.get('GITHUB_CONSUMER_KEY', 'not-set')
 GITHUB_CONSUMER_SECRET = os.environ.get('GITHUB_CONSUMER_SECRET', 'not-set')
 GITHUB_ACCESS_TOKENS = os.environ.get('GITHUB_ACCESS_TOKENS', '').split(',')
 APP_SECRET_KEY = os.environ.get('APP_SECRET_KEY', 'not-set')
+
+# keep disabled authentication by default
+DISABLE_AUTHENTICATION = os.environ.get('DISABLE_AUTHENTICATION', '1')
+if DISABLE_AUTHENTICATION == '1':
+    DISABLE_AUTHENTICATION=False
```
Comment (fridex) [inline f8a_jobs/defaults.py]: `DISABLE_AUTHENTICATION = True`?

Comment (pkajaba) [inline f8a_jobs/defaults.py]: oh, sorry. I mixed it together

Comment (fridex) [inline f8a_jobs/defaults.py]: `DISABLE_AUTHENTICATIO = os.environ.get('DISABLE_AUTHENTICATION', '1') == '1'` for short :)

Comment (jpopelka) [inline f8a_jobs/defaults.py]: `DISABLE_AUTHENTICATION = os.getenv('DISABLE_AUTHENTICATION', '1') == '1'` for even shorter :-D

Comment (pkajaba) [inline f8a_jobs/defaults.py]: Thanks! :+1:

**Pull request merged**